citeproc/model.py

```diff
@@ -373,13 +373,14 @@ def case(self, text, language=None):
                 text = ' '.join(output)
             elif text_case == 'title':
                 output = []
+                # : gurantees that the first word is capitalized
                 prev = ':'
                 for word in text.words():
-                    if not text.isupper() and not word.isupper():
+                    if (str(word) not in self._stop_words or
+                        prev in (':', '.')):
+                        word = word.capitalize_first()
+                    else:
```

Comment:
I am just a fly on the wall here, but I would recommend to
- separate versioneer addition into a separate PR
- accompany PR with code fixes with a test demonstrating/solidifying the need for hte fix

Reply:
FWIW fixing up/extending testing in 
- #137 
so hopefully it would soon gets triggered again on PRs here too

Reply:
Thanks, I've split everything up into independent PRs, and the updated tests will show what's fixed by this PR. I think you need to approve the workflow for the CI to run.